src/core/fullscreenui_settings.cpp

```diff
@@ -4013,7 +4013,7 @@ void FullscreenUI::DrawGraphicsSettingsPage()
                       FSUI_VSTR("Uses PGXP for all instructions, not just memory operations."), "GPU", "PGXPCPU", false,
                       pgxp_enabled);
 
-    DrawToggleSetting(bsi, FSUI_ICONVSTR(ICON_FA_VECTOR_SQUARE, "Vertex Cache"),
+    DrawToggleSetting(bsi, FSUI_ICONVSTR(ICON_FA_CIRCLE_NODES, "Vertex Cache"),
```

Comment:
`VECTOR_SQUARE` was removed in FA7. `CIRCLE_NODES` looks very similar except it's not a square but a triangle.

Reply:
Does the previous commit still compile then? If not, it should be swapped there. I try to have every commit buildable for bisecting.

Reply:
yes, I replaced `VECTOR_SQUARE` in 0dde823aa2204cbf69915612a4234409c7d9d242 together with the version upgrade. All other icon changes are in the following commit and unrelated to the upgrade.